UnityProject/Assets/Scripts/Core/Input System/MouseUtils.cs

```diff
@@ -260,7 +261,9 @@ private static bool IntersectsSprite(SpriteRenderer spriteRenderer, Ray ray, out
 
 		// Craete a plane so it has the same orientation as the sprite transform
 		Plane plane =
-			new Plane(spriteRenderer.transform.forward, (Vector2) spriteRenderer.transform.position); //????????
+			new Plane(spriteRenderer.transform.forward, spriteRenderer.transform.position); //????????
```

Comment:
?????????

Reply:
yeah, (Vector2)  was causing it to lose the z component In resulting in not being able too be clicked when in 3D